zipkin/src/test/java/zipkin/TestObjects.java

```diff
@@ -72,7 +72,6 @@
           .addAnnotation(Annotation.create((TODAY + 150) * 1000, CLIENT_SEND, APP_ENDPOINT))
           .addAnnotation(Annotation.create((TODAY + 200) * 1000, CLIENT_RECV, APP_ENDPOINT))
           .addAnnotation(Annotation.create((TODAY + 190) * 1000, "⻩", NO_IP_ENDPOINT))
-          .addBinaryAnnotation(BinaryAnnotation.address(CLIENT_ADDR, APP_ENDPOINT))
```

Comment:
It is invalid to put a CLIENT_ADDR on a client span. Scrubbed this as it confuses other things